tests/test_restore_names.py

```diff
@@ -1,3 +1,50 @@
+import pytest
 from app.restore_names import restore_names
 
-# write your tests here
+
+@pytest.mark.parametrize(
+    "users, expected_users",
+    [
+        [
+            [{"first_name": None,
```

Comment:
Code is ok, but list of usecases should be formatted correctly, with indentations, for better readability.

Reply:
Exactly